src/libraries/pkg/test/testPackages.proj

```diff
@@ -13,9 +13,13 @@
     <!-- support override via commandline -->
     <RuntimesToInclude Condition="'$(RuntimesToInclude)' != ''" Include="$(RuntimesToInclude)" />
     <TargetFrameworksToInclude Condition="'$(TargetFrameworksToInclude)' != ''" Include="$(TargetFrameworksToInclude)" />
-
+    <SymbolsPackages Include="$(LibrariesAllConfigShippingPackagesDir)*.symbols.nupkg" />
+    <ExcludePackagesList Include="@(SymbolsPackages)" />
+    <ExcludePackagesList Include="@(ExcludePackages->'$(LibrariesAllConfigShippingPackagesDir)%(Identity).$(Version).nupkg')" />
```

Comment:
I don't think you can rely on `$(Version)` here.  During servicing packages will have different versions.   Can you use a wildcard instead?

Reply:
The wildcard has a problem here as i.e, `Microsoft.Extensions.Configuration.*` would exclude all the `Micorosft.Extensions.Configuration` packages right?

Reply:
It would seem so, what do you suggest?

Reply:
I think we could use a wildcard using the MajorProductVersion? All our packages are versioned following the `Major.Minor.Patch` using the same `Major`. So we could do something like:

```xml
<ExcludePackagesList Include="@(ExcludePackages->'$(LibrariesAllConfigShippingPackagesDir)%(Identity).$(MajorVersion)*.nupkg')" />
```
https://github.com/dotnet/runtime/blob/705daa17c134f6485830785a1cd07a27c59261a1/eng/Versions.props#L6

Would that work?

Reply:
WildCards dont work with the transforms, i was able to make it work by union and disjoint of itemgroups, But the solution is complicated, would love to know if there is an easier way to do this.

Reply:
You can change these interfaces if you want, it looks like ExcludePackages is only used internally and PackagesToTest is just a convenience hook for devs who want to run a subset.  If you can achieve the scenarios with different hooks it's probably reasonable.